### ISSUE TITLE
 '%database.host%' should be equal to ... 'localhost'

### ISSUE THREAD

**Issue by petrparolek** [contributte/bootstrap#17]
Probably not working properly

https://github.com/contributte/bootstrap/runs/3398111243?check_suite_focus=true

```
-- FAILED: ExtraConfigurator.phpt
   Failed: '%database.host%' should be equal to 
       ... 'localhost'
   
   in tests/cases/ExtraConfigurator.phpt(138) Assert::equal('localhost', $container->getParameters()['foobar']);
   in nunjuck/src/Toolkit.php(52) Tests\{closure}()
   in tests/cases/ExtraConfigurator.phpt(139) Ninjify\Nunjuck\Toolkit::test()
```

**Comment by f3l1x** [comment]
Yep, thanks. I will take a look.

**Comment by petrparolek** [comment]
@f3l1x ping

**Comment by f3l1x** [comment]
Solved 65d7cb1f2654614639ba7a569e4f401ec373ae1f
